src/change_stream/mod.rs

```diff
@@ -132,3 +157,36 @@ impl Stream for ChangeStream {
         todo!();
     }
 }
+
+// Specification used to create a new changeStream.
```

Comment:
can we rewrite "changeStream" as `ChangeStream` (with backticks) to emphasize its the type we're referring to?

Reply:
fixed